src/Relude/Monad/Either.hs

```diff
@@ -112,46 +112,86 @@ maybeToLeft :: r -> Maybe l -> Either l r
 maybeToLeft r = maybe (Right r) Left
 {-# INLINE maybeToLeft #-}
 
--- | Applies given action to 'Either' content if 'Left' is given and returns
+{- | Applies given action to 'Either' content if 'Left' is given and returns
 -- the result. In case of 'Right' the default value will be returned.
+
+>>> whenLeft "bar" (Left "foo") (\a -> [a])
+["foo"]
+
+>>> whenLeft "bar" (Right "foo") (\a -> [a])
+["bar"]
+
+-}
 whenLeft :: Applicative f => a -> Either l r -> (l -> f a) -> f a
 whenLeft _ (Left  l) f = f l
 whenLeft a (Right _) _ = pure a
 {-# INLINE whenLeft #-}
 
--- | Applies given action to 'Either' content if 'Left' is given.
+{- | Applies given action to 'Either' content if 'Left' is given.
+
+TODO: whenLeft_
```

Comment:
Here you can use `putTextLn` function as an argument to `whenLeft_`

Reply:
I've tried various combinations I can't quite work out what you mean?

Reply:
I mean something like this:

```haskell
>>> whenLeft_ (Right 42) putTextLn
>>> whenLeft_ (Left "foo") putTextLn
foo
```